internal/resources/kustomization/resource_kustomization_test.go

```diff
@@ -34,7 +36,6 @@ const (
 	kustomizationResourceVar   = "test_kustomizationy"
 	kustomizationDataSourceVar = "test_data_source_kustomizationy"
```

Comment:
Type: test_data_source_kustomizationy ? and on L36?

Reply:
updated to test_kustomization PTAL

Reply:
updated to test_kustomization PTAL